### ISSUE TITLE
how to change background colors?  

### ISSUE THREAD

**Issue by suberlak** [Bash-it/bash-it#958]
Hi! 
According to https://github.com/Bash-it/bash-it/wiki/Themes   , each theme has more than 1 background color available. I studied README, and anything I could find, but still have no idea how to change the theme background color.  It should be possible, but how to do it with bash_it  ? Also, how do you change any other colors in a theme ? 
Best wishes
Chris

**Comment by nwinkler** [comment]
You can change your background color in your terminal app. On macOS for example, you can select a _Profile_ in the Terminal.app's Preferences screen. The profile defines the background color (and some of the foreground/highlight colors as well). Other terminal apps have similar functionality.

**Comment by suberlak** [comment]
Thank you for your note,  I figured it out thanks to my officemate soon after posting the question.  But perhaps someone else will have that question in the future - perhaps a note about this can be added to the description of how to change themes?  It seems obvious once I know, but before that I was quite confused about what to do to make the profile look  as in the examples provided. ( that's why I posted the question).  Thank you for your work , it is a great set of themes! 
Best wishes,
Chris
